pom.xml

```diff
@@ -678,6 +710,7 @@
             <properties>
                 <maven.jar.version>3.2.2</maven.jar.version>
                 <maven.compiler.version>3.8.1</maven.compiler.version>
+                <felix.version>3.5.1</felix.version>
```

Comment:
3.5.1 is known to have some RB issues: why not upgrade to 5.1.9?

Reply:
We still support JDK 7, so that's probably it.

Reply:
yeah 😭, when testing with Java 7 we use the older version, when testing with Java 8+ we use the latest.

We should probably use toolchains for testing with Java 7, but... it's getting dropped soon, so this is a temporary hack.